### ISSUE TITLE
Deactivate by default `Exclude all upper-case variables` option in Variable Explorer

### ISSUE THREAD

**Issue by texadactyl** [spyder-ide/spyder#16402]
![spyder_version](https://user-images.githubusercontent.com/11318756/132954925-30bdcedb-beec-47a0-b753-cf7bc7ab96b2.png)

Program source:
```
C = 3e8 # m/s
CSQ = C * C
csq = C * C
D = C + 1
DSQ = D * D
dsq = D * D
E = D + 1
```

This is what I see in the Variable Explorer:
![ve](https://user-images.githubusercontent.com/11318756/132954980-7c9db6ad-f199-46d3-b767-6bbaa868e89a.png)

Why are some of the variables whose names are spelled in all caps not displayed?

**Comment by ccordoba12** [comment]
Hey @texadactyl, thanks for reporting. You need to deactivate the option called `Exclude all upper-case variables` in the Options menu of the Variable Explorer:

![imagen](https://user-images.githubusercontent.com/365293/132959687-be7e71a4-929a-4f6a-80ef-319adccfc9b6.png)

I think having that option activated by default is counterintuitive and it usually surprises people the wrong way (like in this case). @texadactyl, what do you think? Should we deactivate it?

**Comment by texadactyl** [comment]
Thanks, @ccordoba12 
That really was a surprise.  Works fine now.
Yes, I think that it should be deactivated by default for the reason you said - counterintuitive.

**Comment by ccordoba12** [comment]
> Yes, I think that it should be deactivated by default for the reason you said - counterintuitive.

Ok, we'll fix that in our next version (5.2.0), to be released next month.
